js/data/bucket.js

```diff
@@ -81,7 +81,7 @@ function Bucket(options) {
  * @private
  */
 Bucket.prototype.addFeatures = function() {
-    for (var i = 0; i < this.features.length; i++) {
+    for (var i = this.features.length - 1; i >= 0; i--) {
```

Comment:
I will spend some more time trying to understand this, I don't have an immediate explanation. This is what I had to do to make test-suite pass :grimacing: